### PR TITLE
Loosen the version constraint for clang-tidy

### LINKED ISSUE #2152: [Clang-Tidy] Missing package
The Docker build for Clang-Tidy fails now:

```console
$ bundle exec rake docker:build ANALYZER=clang_tidy
...
#7 3.789 Get:4 https://apt.llvm.org/buster llvm-toolchain-buster-10 InRelease [4,241 B]
#7 4.046 Get:5 https://apt.llvm.org/buster llvm-toolchain-buster-10/main amd64 Packages [14.7 kB]
#7 4.087 Fetched 19.0 kB in 1s (16.0 kB/s)
#7 4.087 Reading package lists...
#7 5.449 E: Version '1:10.0.1~++20200708124224+ef32c611aa2-1~exp1~20200707224822.188' for 'clang-tidy-10' was not found
------
executor failed running [/bin/sh -c curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - &&     add-apt-repository "deb http://apt.llvm.org/buster/ llvm-toolchain-buster-10 main" &&     apt-get update &&     grep -v "^\s*#" "${RUNNER_USER_HOME}/packages.list" | xargs apt-get install -qq -y --no-install-recommends &&     apt-get clean &&     update-alternatives --install /usr/local/bin/clang-tidy clang-tidy /usr/lib/llvm-10/bin/clang-tidy 20]: exit code: 123
rake aborted!
Command failed with status (1): [docker build --tag sider/runner_clang_tidy...]
...
```

### PR DESCRIPTION
**Pull request by yykamei**
> Explain a summary, purpose, or background for this change.

At the time when choosing the apt(8) source, the major version of clang-tidy is fixed to `10`, so I loosened the version constraint in `packages.list`. In my opinion, it's sufficient for us to care about only the major version of LLVM.

http://blog.llvm.org/2016/12/llvms-new-versioning-scheme.html

> Refer related issues or pull requests (e.g. `Close #123` or `None`).

Fix #2152

> Check the following if needed.

- [x] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

**Comment by yykamei** [comment]
What do you think about my opinion?

> In my opinion, it's sufficient for us to care about only the major version of LLVM.

File: images/clang_tidy/packages.list [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Main Clang-Tidy package
-clang-tidy-10=1:10.0.1~++20200708124224+ef32c611aa2-1~exp1~20200707224822.188
```
Comment (ybiquitous) [inline images/clang_tidy/packages.list:2]: The new version: `1:10.0.1~++20210308123532+ef32c611aa21-1~exp1~20210308114140.189`

https://github.com/sider/runners/pull/2153/checks?check_run_id=2063521534#step:6:257

**Review by ybiquitous — APPROVED** [review]
It looks reasonable to me 👍

**Comment by yykamei** [comment]
Oh, I remember @suetsuguk is off today. Sorry for mentioning :pray:

**Comment by yykamei** [comment]
Thanks, I merge this PR.

**Comment by ybiquitous** [comment]
Thanks. I've reverted the commit status of `clang_tidy` to *required*.

**Pull request merged**